spinoso-time/src/time/tzrs/convert.rs

```diff
@@ -1,12 +1,11 @@
-use core::fmt::{Display, Formatter, Result};
+use core::fmt;
 
 use super::{Time, ToA};
 
-impl Display for Time {
+impl fmt::Display for Time {
     /// Returns a canonical string representation of _time_.
     ///
-    /// Can be used to implement the Ruby method [`Time#asctime`],
-    /// [`Time#ctime`], [`Time#to_s`], and [`Time#inspect`].
+    /// `Display` uses the same format as [`Time::to_s`].
```

Comment:
```suggestion
    /// `Display` uses the same format as [`Time#to_s`].
```

:point_up: Needs to be the same as the rustdoc ref right?

Reply:
derp thank you.